openlibrary/macros/Pager.html

```diff
@@ -1,6 +1,6 @@
 $def with (page, num_found, results_per_page=20)
 
-$ pages = (num_found // results_per_page) + 1
+$ pages = (num_found // results_per_page) + (1 if (num_found % results_per_page) > 0 else 0)
```

Comment:
We should be using ceil here :P Add it to the template globals (see https://github.com/internetarchive/openlibrary/blob/10eba420189c966f5ac63c871681460a92f90972/openlibrary/plugins/openlibrary/code.py#L878 ) and then call it like so: `ceil(num_found / results_per_pager)` . You'll also need to remove the `+1` in the for loop below.

Reply:
`ceil()` will definitely be cleaner :D !
`range()` upper bound is not included so we do need the `+1` to reach the `last_page_in_set`.

Reply:
@cdrini, should I import the whole `math` module or just the `ceil()` function in `code.py`? What is the preferred method?